src/main/java/org/opensearch/security/ssl/DefaultSecurityKeyStore.java

```diff
@@ -968,18 +968,26 @@ private SslContext buildSSLServerContext(
         final ClientAuth authMode
     ) throws SSLException {
 
-        final SslContextBuilder _sslContextBuilder = configureSSLServerContextBuilder(
-            SslContextBuilder.forServer(_key, _cert),
-            sslProvider,
-            ciphers,
-            authMode
-        );
+        try {
```

Comment:
Refactor 'doPrivileged(...) with try catch into a function so you can rewrite this block as the following?
```java
final SslContextBuilder _sslContextBuilder = this.doPrivilegedSslAction(() ->
   configureSSLServerContextBuilder(SslContextBuilder.forServer(_key, _cert), sslProvider, ciphers, authMode));
```